dracut/99start-root/module-setup.sh

```diff
@@ -5,6 +5,5 @@
 install() {
     inst_rules "$moddir/65-start-root.rules"
 
-    rm -f "$initdir/usr/lib/systemd/system/systemd-tmpfiles-setup.service"
-    rm -f "$initdir/usr/lib/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup.service"
+    sed -i 's/After=local-fs.target /After=/' "$initdir/usr/lib/systemd/system/systemd-tmpfiles-setup.service"
```

Comment:
A bit more future proof would be something like that to remove `local-fs.target` and any space before and after it, so that it doesn't directly have to be after the `After=` with an element behind it but one also has to avoid removing it if it is the only occurrence in the line because `After=` would reset the dependencies specified before, or? For now I think it's fine and when it breaks it will be easy to adjust.